dev-haskell/apply-refact/apply-refact-0.14.0.0.ebuild

```
@@ -0,0 +1,40 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.5.0.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Perform refactorings specified by the refact library"
+HOMEPAGE="https://github.com/mpickering/apply-refact"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+
+RDEPEND=">=dev-haskell/extra-1.7.3[profile?]
+	>=dev-haskell/filemanip-0.3.6.3[profile?] <dev-haskell/filemanip-0.4[profile?]
+	dev-haskell/ghc-exactprint[profile?]
+	dev-haskell/ghc-paths[profile?]
+	>=dev-haskell/optparse-applicative-0.15.1.0[profile?]
+	>=dev-haskell/refact-0.2[profile?]
+	>=dev-haskell/syb-0.7.1[profile?]
+	>=dev-haskell/uniplate-1.6.13[profile?]
+	>=dev-haskell/unix-compat-0.5.2[profile?]
+	>=dev-lang/ghc-9.2[profile?] <dev-lang/ghc-9.3[profile?]
+	>=dev-lang/ghc-9.2.4
+	|| ( ( >=dev-haskell/ghc-exactprint-1.5.0[profile?] <dev-haskell/ghc-exactprint-1.6[profile?] )
+		|| ( ( >=dev-haskell/ghc-exactprint-1.6.0[profile?] <dev-haskell/ghc-exactprint-1.7[profile?] )
+			|| ( ( >=dev-haskell/ghc-exactprint-1.7.0[profile?] <dev-haskell/ghc-exactprint-1.8[profile?] )
+			( >=dev-haskell/ghc-exactprint-1.8.0[profile?] <dev-haskell/ghc-exactprint-1.9[profile?] ) ) ) )
```

`hackport` doesn't generate this section correctly although I have been working on a patch to fix this (see e.g. https://github.com/gentoo-haskell/hackport/issues/58). Here, you can see that the version ranges are contiguous, so it can be simplified to avoid the CI error:

```suggestion
	>=dev-haskell/ghc-exactprint-1.5:=[profile?] <dev-haskell/ghc-exactprint-1.9:=[profile?]
```